framework/wazuh/core/engine/events.py

```diff
@@ -17,4 +21,19 @@ async def send(self, events: List[StatelessEvent]) -> None:
         events : List[StatelessEvent]
             Events list.
         """
-        # TODO(25121): Send events to the engine once the API endpoint is available.
+        try:
+            for event in events:
+                response = await self._client.post(
+                    url=f'{self.API_URL}/{self.MODULE}',
+                    json=asdict(event, dict_factory=convert_enums),
```

Comment:
Since the StatelessEvent model does not contain enay enum, we can avoid specifying the dictionary factory to avoid doing unnecessary operations

```suggestion
                    json=asdict(event),
```